pycbc/population/rates_functions.py

```diff
@@ -10,7 +10,7 @@
 import bisect
 from pycbc.conversions import mchirp_from_mass1_mass2
 
-def process_full_data(fname, rhomin, mass1, mass2, lo_tlt_mass, hi_tlt_mass):
+def process_full_data(fname, rhomin, mass1, mass2, lo_mchirp, hi_mchirp):
     """Read the zero and time-lagged triggers identified by BBH templates.
```

Comment:
Change 'BBH templates' to 'templates in a specified range of chirp mass'

Reply:
done! thanks